Tests/test_imagecolor.py

```diff
@@ -6,32 +6,96 @@
 
 class TestImageColor(PillowTestCase):
 
-    def test_sanity(self):
+    def test_hash(self):
         self.assertEqual((255, 0, 0), ImageColor.getrgb("#f00"))
-        self.assertEqual((255, 0, 0), ImageColor.getrgb("#ff0000"))
+        self.assertEqual((0, 255, 0), ImageColor.getrgb("#0f0"))
+        self.assertEqual((0, 0, 255), ImageColor.getrgb("#00f"))
```

Comment:
Maybe add some uppercase here and later? e.g.
`self.assertEqual((0, 0, 255), ImageColor.getrgb("#00F"))`